### PR TITLE
Feature/58 login logout

### PR DESCRIPTION
**Pull request by naitoarekkus**
Clients can now create themselves (persistence is not implemented)
They can login and get a token.
They can logout and invalidate their token

### CODE REVIEW

File: WebService/Models/Token.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿namespace WebService.Models
+{
+    public class Token
+    {
+        public string Tkn { get; set; }
+
+        public Token(string token)
+        {
+            Tkn = token;
```
Comment (MikaelVM) [inline WebService/Models/Token.cs:9]: Lille ting, "Tkn" opfuskere hvad variablen er til andre steder den bliver nævnt, så rammer readability en lille bitte del.

File: WebService-UnitTests/ContentReaderTest.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+using Xunit;
+
+namespace WebService_UnitTests
+{
+    public class ContentReaderTest
+    {
+        [Fact]
+        public void ReadStreamContent_WithDefaultEncoding_StreamWithContentUsingDefaultEncoding_ReturnsCorrectString()
+        {
+            // Steup
```
Comment (Kaliahh) [inline WebService-UnitTests/ContentReaderTest.cs:14]:
```suggestion
            // Setup
```

File: WebService-UnitTests/ContentReaderTest.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            // Act
+            string actual = ContentReader.ReadStreamContent(stream);
+            
+            // Assert
+            Assert.NotEqual(original, actual);
+        }
+        
+        [Fact]
+        public void
+            ReadStreamContent_WithUnicodeEncoding_StreamWithContentUsingSameEncoding_ReturnsIncorrectString()
```
Comment (Kaliahh) [inline WebService-UnitTests/ContentReaderTest.cs:59]:
```suggestion
            ReadStreamContent_WithUnicodeEncoding_StreamWithContentUsingSameEncoding_ReturnsCorrectString()
```

File: WebService/Models/Token.cs (hunk 1 above)
Comment (Kaliahh) [reply WebService/Models/Token.cs:9]: ++

**Review by Kaliahh — CHANGES_REQUESTED** [review]
Småting

**Review by MikaelVM — DISMISSED** [review]

**Review by Kaliahh — APPROVED** [review]

**Review by MikaelVM — APPROVED** [review]
Seems good.

**Pull request merged**